tests/README.md

```diff
@@ -144,5 +154,47 @@ Bugzilla is about `sshd_config` being not altered correctly. Thus:
 Now, you can perform validation check with command
 
 ```
-./test_suite.py rule --hypervisor qemu:///system --domain ssg-test-suite-centos --datastream ./ssg-centos7-ds.xml rule_sshd_disable_kerb_auth
+./test_suite.py rule --libvirt qemu:///system ssg-test-suite-centos --datastream ./ssg-centos7-ds.xml rule_sshd_disable_kerb_auth
+```
+
+## Docker backend
+
+You can also use the docker-based for running tests, just use the script with the `--docker` argument.
+You need to provide `--docker <base image name>` option on the command-line.
+
+To obtain the base image, you can use `test_suite-*` Dockerfiles in the `Dockerfiles` directory to build it.
+We recomend to use RHEL-based containers, as the test suite is optimized for testing the RHEL content.
+
+Build the image using this command:
+
+```
+docker build --build-arg CLIENT_PUBLIC_KEY="ssh-rsa AAAAB3NzaC1y...rJSs4BL me@localhost" -t ssg_test_suite -f test_suite-rhel .
+```
+
+Run the test suite using this command:
+
+```
+./test_suite.py rule --docker ssg_test_suite --datastream ./ssg-centos7-ds.xml rule_sshd_disable_kerb_auth
```

Comment:
This command does not work, because --xccdf-id is required argument.

Also, the test_suite.py is not usually located in same directory as built DSs. Instead, the DSs can be found in `../build`